Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Hack to fix awk call.

diff -Naurp plymouth.orig/scripts/plymouth-set-default-theme.in plymouth/scripts/plymouth-set-default-theme.in
--- plymouth.orig/scripts/plymouth-set-default-theme.in
+++ plymouth/scripts/plymouth-set-default-theme.in
@@ -45,15 +45,7 @@ function list_themes ()
 function read_theme_name_from_file ()
 {
         echo $(grep -v '^#' $1 2> /dev/null |
-               awk '
-                     BEGIN {
-                        RS="[[][[:blank:]]*[^[:space:]]+[:blank:]*[]\n]";
-                        FS="[=[:space:]]+";
-                        OFS="";
-                        ORS=""
-                     }
-                     $1 ~/Theme/ { print $2 }
-               ')
+		awk -F= '/Theme=/ { print $2 }')
 }
 
 function get_default_theme ()
